rllib/core/rl_module/rl_module.py

```diff
@@ -277,6 +277,13 @@ class RLModule(abc.ABC):
 
     framework: str = None
 
+    # Constants that signify forward pass modes.
```

Comment:
enum these plz.

Reply:
and don't forget to update the docstrings accordingly.

Reply:
As per our offline discussion, the getter will be split up between three get methods and thus eliminate the need for the enumeration. The tradeoff being that if one wants to inherit from RLModule, they must implement three methods but there is no need for such constants to be used when getting action dist classes.